src/app/app.component.ts

```diff
@@ -1,51 +1,49 @@
-import { Component, ViewEncapsulation,OnInit } from '@angular/core';
+import { Component, ViewEncapsulation, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
 import { NgbModal, NgbModalConfig } from '@ng-bootstrap/ng-bootstrap';
 import { AuthService } from './auth/auth.service';
+
 @Component({
   selector: 'app-root',
   encapsulation: ViewEncapsulation.None,
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css'],
 })
-export class AppComponent implements OnInit {
+export class AppComponent {
   modal: any;
   profileModal: any;
-  isMenuActive: boolean = false;
-  isActive: boolean = false;
-  user;
-
+  isMenuActive = false;
+  isActive = false;
   constructor(
     public authService: AuthService,
     config: NgbModalConfig,
     private modalService: NgbModal,
     private profileModalService: NgbModal,
+    public router: Router
   ) {
     config.backdrop = 'static';
   }
-  open(content) {
+  open(content: any): void {
     this.modal = this.modalService.open(content, {
```

Comment:
We can enable a rule here called strict checks.Actually using `any` as a type is a bad practice i.e why it is called Typescript ,we can get delay in getting response and some console errors with use of `any`.Should we enable strict checks at this point or after completion of the project ?WDYT?